sweep/sweeper.go

```diff
@@ -956,25 +955,24 @@ func (s *UtxoSweeper) clusterByLockTime(inputs pendingInputs) ([]inputCluster,
 		}
 
 		input.lastFeeRate = feeRate
-		inputFeeRates[op] = feeRate
 	}
 
 	// We'll then determine the sweep fee rate for each set of inputs by
```

Comment:
the plan here is to remove the locktime field - we track the spending condition outside the sweeper, the caller won't send a sweep request unless the input has been mature, so this locktime field has no utility here.

Reply:
understand, tho you have to be careful, because send-level htlcs commit to the exact locktime whereas there are situations where other outputs commit to a `greater than` locktime. So both will be offered to the sweeper when they are ready to be spent, tho you have to be careful because you cannot mix them in some situations.